lms/lmsweb/views.py

```diff
@@ -100,7 +100,8 @@ def login(login_error: str = None):
             login_error = 'Invalid username or password'
             return redirect(
                 url_for(
-                    'login', **{'next': next_page, 'login_error': login_error}),
+                    'login', **{'next': next_page,
+                                'login_error': login_error}),
```

Comment:
I'd prefer:
`return redirect(url_for(
    `your parameters`,
    `other lines of parameters if needed`,
))

Reply:
thanks, I appreciate the suggestion. I think Yam told me last time he prefers some other format so I'll wait for his input on this one

Reply:
I think the best solution for this one is:

```python
error_details = {'next': next_page, 'login_error': login_error}
return redirect(url_for('login', **error_details))